estate/models/estate_user.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class EstateUser(models.Model):
```

Comment:
```suggestion
class ResUser(models.Model):
```

The file should also be called res_users. this allows to see instatly form the filename that you are inheriting res users